src/config/passportConfig.js

```diff
@@ -0,0 +1,38 @@
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { User } from '../models/index';
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((user, done) => {
+  done(null, user.id);
+});
+
+const passportConfig = () => {
+  passport.use('local-login', new LocalStrategy(
+    {
+      usernameField: 'email',
+      passwordField: 'password',
+      passReqToCallback: true
+    },
+    (async (req, email, password, done) => {
+      try {
+        const user = await User.findOne({ where: { email } });
+        if (!user) {
+          return done(null, false, { message: 'Incorrect credentials' });
+        }
+        if (password !== user.password) {
```

Comment:
Just so you know, we will not be storing a plain password in the db. Seed the database with a hashed password and use the compare method we attached to the user model prototype. This will always fail when local signup is ready